engine/dlib/src/dlib/log.cpp

```diff
@@ -340,7 +340,7 @@ static void DoLogSynchronized(LogSeverity severity, const char* domain, const ch
         g_Listeners[i]((LogSeverity)severity, domain, output);
     }
 
-    dmProfile::LogText("%s", output);
+    ProfileLogText("%s", output);
```

Comment:
Going more C-style helps with generating api's for other languages, but It also feels like a good choice for an api.